src/model/article-vor.v6.yaml

```diff
@@ -101,5 +119,3 @@ allOf:
               - content
     required:
       - body
-    dependencies:
-        authorResponse: [decisionLetter]
```

Comment:
If I understand this `dependencies` block correctly it means that where there is an `authorResponse` property set we should require a `decisionLetter` to be set?

Should we be removing this entirely or do we need to express a more complex `dependencies` block which states that if an `authorResponse` is present would should expect one of `decisionLetter` or `editorEvaluation`?

Reply:
I looked briefly at JSON schema to see how you might be able to specify a "this or that" structure and I didn't find anything quickly. If you can figure out how to satisfy the the complexity it would be welcome I think. Do you anticipate the removal of the dependencies to cause any programming challenges if we had to remove it?

Reply:
> Do you anticipate the removal of the dependencies to cause any programming challenges if we had to remove it?

No, but it's about reflecting in the schema, as much as possible, the expectations of the content properties. I don't want to lose what we already have in place here. Currently, if we have JSON that includes an `authorResponse` but not a `decisionLetter` then this issue would be flagged and we could fix it. I will try and work on this change todo and we could add a couple more samples to prove this.